tests/validators_test.py

```diff
@@ -93,3 +91,84 @@ def test_is_file_valid(self, source, target, answer, validator):
+            copyfile(source_release_notes, target_release_notes)
+            patch.object(ReleaseNotesValidator, 'get_master_diff', side_effect=self.mock_get_master_diff)
+            validator = ReleaseNotesValidator(target_dummy)
+            assert validator.validate_file_release_notes() is answer
+        finally:
+            os.remove(target_dummy)
+            os.remove(target_release_notes)
+
+    @staticmethod
+    def create_release_notes_structure_test_package():
+        changelog_needed = [
+            (VALID_SCRIPT_PATH, 'Script'),
+            (VALID_INTEGRATION_TEST_PATH, 'Integration')
+        ]
+
+        changelog_files_answer = [
+            (VALID_ONE_LINE_CHANGELOG_PATH, True),
+            (VALID_ONE_LINE_LIST_CHANGELOG_PATH, True),
+            (VALID_MULTI_LINE_CHANGELOG_PATH, True),
+            (VALID_MULTI_LINE_LIST_CHANGELOG_PATH, True),
+            (INVALID_ONE_LINE_1_CHANGELOG_PATH, False),
+            (INVALID_ONE_LINE_2_CHANGELOG_PATH, False),
+            (INVALID_ONE_LINE_LIST_1_CHANGELOG_PATH, False),
+            (INVALID_ONE_LINE_LIST_2_CHANGELOG_PATH, False),
+            (INVALID_MULTI_LINE_1_CHANGELOG_PATH, False),
+            (INVALID_MULTI_LINE_2_CHANGELOG_PATH, False)
+        ]
+
+        test_package = list()
+
+        for (dummy_file, file_type) in changelog_needed:
+            for (release_notes_file, answer) in changelog_files_answer:
+                if file_type == 'Script':
+                    test_package.append((dummy_file, SCRIPT_TARGET, release_notes_file,
+                                         SCRIPT_RELEASE_NOTES_TARGET, ReleaseNotesValidator, answer))
+                elif file_type == 'Integration':
+                    test_package.append((dummy_file, INTEGRATION_TARGET, release_notes_file,
+                                         INTEGRATION_RELEASE_NOTES_TARGET, ReleaseNotesValidator, answer))
+
+        return test_package
+
+    test_package = create_release_notes_structure_test_package.__func__()
+
+    @pytest.mark.parametrize('source_dummy, target_dummy, source_release_notes, target_release_notes, '
+                             'validator, answer', test_package)
+    def test_valid_release_notes_structure(self, source_dummy, target_dummy,
+                                           source_release_notes, target_release_notes, validator, answer):
+        # type: (str, str, str, str, Type[BaseValidator], Any) -> None
+        try:
+            copyfile(source_dummy, target_dummy)
+            copyfile(source_release_notes, target_release_notes)
+            patch.object(ReleaseNotesValidator, 'get_master_diff', side_effect=self.mock_get_master_diff)
+            validator = ReleaseNotesValidator(target_dummy)
+            assert validator.is_valid_release_notes_structure() is answer
+        finally:
+            os.remove(target_dummy)
+            os.remove(target_release_notes)
+
+    @staticmethod
+    def mock_get_master_diff():
```

Comment:
shouldn't this be changed everytime?

Reply:
The `master_diff` is not relevant for this test, but we have to mock it for the test to work as expected.

Reply:
so when is it relevant?

Reply:
Only to see whether a new comment was added. This test checks only the structure of the changes.